config/routes.config.js

```diff
@@ -43,6 +43,8 @@ const routes = [
   { path: '/trillium/longTermCare/type', editInfo: 'deductions.trilliumLongTermCareTypeClaim' },
   { path: '/trillium/longTermCare/type/amount', editInfo: 'deductions.trilliumLongTermCareAmount' },
   { path: '/deductions/climate-action-incentive' },
+  { path: '/vote/optIn' },
+  { path: '/vote/confirmRegistration', editInfo: 'vote.voterPageEdited' },
```

Comment:
Is there a way to use another one of the keys instead of this "edited" key?

Reply:
I played around with that for awhile - the issue is we want it to show up if either of them have been ticked. Might be possible to do it for whether they ticked yes or no on the previous page?

Reply:
The existing code doesn't support using the Yes / No from the optIn page to trigger this - it's just looking for a non-null value (which it will be even if set to 'No'). We can either refactor to support this use case, or keep it the way it is in this PR.

Reply:
What about if we only asked the first one? 
(If they're not a Canadian citizen, we don't let them vote, right?)

Reply:
Both questions need to be answered yes for them to be registered, if they only tick the second one we still want this page to show up.

Reply:
I think we're pretty deep in edge-case land, but I defer to your judgement since it seems pretty low-effort overall.

Reply:
I agree, this is fairly edge case. If it becomes a problem we can revisit. Changed to voterCitizen